core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -2399,6 +2399,21 @@ private void checkPredicates(RelOptCluster cluster, RelOptTable empTable,
     assertThat(RelMdUtil.linear(12, 0, 10, 100, 200), is(200d));
   }
 
+  // ----------------------------------------------------------------------
+  // Tests for getExpressionLineage
+  // ----------------------------------------------------------------------
+
+  private void assertExpressionLineage(
+      String sql, int columnIndex, String expected, String comment) {
+    RelNode rel = convertSql(sql);
+    RelMetadataQuery mq = rel.getCluster().getMetadataQuery();
+    RexNode ref = RexInputRef.of(columnIndex, rel.getRowType().getFieldList());
+    Set<RexNode> r = mq.getExpressionLineage(rel, ref);
+
+    assertEquals(expected, String.valueOf(r), "Lineage for expr '" + ref + "' in node '"
+        + rel + "'" + " for query '" + sql + "':n" + comment);
```

Comment:
`:n`?

Reply:
Sorry about that, I lost a character in the refactoring and I did not notice.

Reply:
Actually it was not my doing, it was `./gradlew autostyleApply` doing that:

```
     assertEquals(expected, String.valueOf(r), "Lineage for expr '" + ref + "' in node '"
-        + rel + "'" + " for query '" + sql + "':\n" + comment);
+        + rel + "'" + " for query '" + sql + "':n"
+  + comment);
   }
```

Looks like a bug to me.

Reply:
https://github.com/apache/calcite/blob/fd3c1d0bdf907d192d42c18b3aff5306eec650e7/build.gradle.kts#L542 might be relevant

Reply:
I have removed the `\n` to avoid issues and filed https://github.com/autostyle/autostyle/issues/30

Reply:
`"\\n\"\n  +"` should probably be `"\\\\n\"\n  +"`

Reply:
@asolimando , I believe the issue has little to do with autostyle, and it seems to be the issue with Calcite's customization.

Please check Calcite-specific regex replacements (see build.gradle.kts#L542)